<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SolidActions</class>
 <widget class="QWidget" name="SolidActions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>384</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTreeView" name="TvActions"/>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="GlActions">
     <item row="0" column="0">
      <widget class="KPushButton" name="PbAddAction">
       <property name="text">
        <string>Add...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KPushButton" name="PbEditAction">
       <property name="text">
        <string>Edit...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="KPushButton" name="PbDeleteAction">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
